content/pages/meetup/2024/0911-nycu.rst

```diff
@@ -0,0 +1,125 @@
+========================================
+Meetup 2024 September 11th in NYCU
+========================================
+
+:date: 2024-09-07 13:42
+:url: meetup/2024/0911-nycu
+:save_as: meetup/2024/0911-nycu.html
+
+Date
+-----
+
+* Date: September 11th, 2024
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+There will be many different subjects discussed at the same time.
+
++-----------------+--------------------------+---------------------------------+
+| Timetable       | Topic 1                  | Topic 2                         |
++=================+==========================+=================================+
+| 18:30 ~ 19:30   | free chat                                                  |
++-----------------+--------------------------+---------------------------------+
+| 19:30 ~ 21:00   | free chat                | modmesh discussion and training |
++-----------------+--------------------------+---------------------------------+
+| 21:00 ~ 21:30   | free chat                                                  |
++-----------------+--------------------------+---------------------------------+
+
+
+|
+
+Subjects
+------------------
+
+modmesh discussion and training
```

Comment:
It looks good to keep both discussion and training together.  Thanks, @ThreeMonth03 .